### PR TITLE
Create Dispatcher model

### PR DESCRIPTION
**Pull request by DmitrS**

### CODE REVIEW

File: app/models/dispatcher.rb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+class Dispatcher < ApplicationRecord
+  validates :name, presence: true
+  validetes :emila, presence: true
```
Comment (rakvium) [inline app/models/dispatcher.rb]: @DmitrS `validetes`? `emila`? :trollface: 
Please fix typos :memo:

Comment (rakvium) [inline app/models/dispatcher.rb]: @DmitrS by the way, what do you think about email uniqueness?

File: app/models/dispatcher.rb [hunk 2]
```diff
@@ -0,0 +1,5 @@
...
+  validetes :password, presence: true
```
Comment (rakvium) [inline app/models/dispatcher.rb]: @DmitrS > you have a typo in `validetes` here too :memo:

File: .gitignore [hunk 3]
```diff
@@ -24,4 +24,4 @@ capybara-*.html
 !/tmp/.keep
 
 # Ignore Byebug command history file.
-.byebug_history
+.byebug_history
```
Comment (rakvium) [inline .gitignore]: Please add a newline to the end of the file :construction_worker_man:

File: app/models/dispatcher.rb [hunk 4]
```diff
@@ -0,0 +1,5 @@
+class Dispatcher < ApplicationRecord
+  validates :name, presence: true
+  validates :email, presence: true, uniqueness: true
+  validates :password, presence: true
+end
```
Comment (rakvium) [inline app/models/dispatcher.rb]: @DmitrS please add a newline to the end of the file :construction_worker_man:

You can config Sublime to do it automatically for you: https://forum.sublimetext.com/t/make-saving-newline-at-eof-the-installation-default/9842/2

File: db/migrate/20161206130507_create_dispatchers.rb [hunk 5]
```diff
@@ -0,0 +1,14 @@
+class CreateDispatchers < ActiveRecord::Migration[5.0]
+  def change
+    create_table :dispatchers do |t|
+      t.string :email
+      t.string :string
```
Comment (rakvium) [inline db/migrate/20161206130507_create_dispatchers.rb]: What these `t.string :string`s for?

File: db/migrate/20161206130507_create_dispatchers.rb [hunk 6]
```diff
@@ -0,0 +1,14 @@
...
+      t.string :password
+      t.string :string
```
Comment (rakvium) [inline db/migrate/20161206130507_create_dispatchers.rb]: What these `t.string :string`s for?

File: db/migrate/20161206130507_create_dispatchers.rb [hunk 7]
```diff
@@ -0,0 +1,14 @@
...
+      t.string :name
+      t.string :string
```
Comment (rakvium) [inline db/migrate/20161206130507_create_dispatchers.rb]: What these `t.string :string`s for?

File: db/migrate/20161206130507_create_dispatchers.rb [hunk 8]
```diff
@@ -0,0 +1,14 @@
+class CreateDispatchers < ActiveRecord::Migration[5.0]
+  def change
+    create_table :dispatchers do |t|
+      t.string :email
```
Comment (rakvium) [inline db/migrate/20161206130507_create_dispatchers.rb]: @DmitrS so what about to add validations in the database too?
http://stackoverflow.com/questions/13122791/rails-validation-in-model-vs-migration

File: test/fixtures/dispatchers.yml [hunk 9]
```diff
@@ -0,0 +1,11 @@
+# Read about fixtures at http://api.rubyonrails.org/classes/ActiveRecord/FixtureSet.html
+
+one:
+  email: MyString
+  password: MyString
+  name: MyString
+
+two:
+  email: MyString
```
Comment (rakvium) [inline test/fixtures/dispatchers.yml]: @DmitrS this fixture won't work, as far as email should be unique.

**Comment by rakvium** [comment]
@DmitrS please fix conflicts.

File: db/migrate/20161206130507_create_dispatchers.rb [hunk 10]
```diff
@@ -0,0 +1,12 @@
+class CreateDispatchers < ActiveRecord::Migration[5.0]
+  def change
+    create_table :dispatchers do |t|
+      t.string :email, :null => false
```
Comment (rakvium) [inline db/migrate/20161206130507_create_dispatchers.rb]: [Use the new Ruby 1.9 hash syntax.](https://github.com/bbatsov/ruby-style-guide#hash-literals)

Comment (DmitrS) [inline db/migrate/20161206130507_create_dispatchers.rb]: Done commit 182a6a0bd4b52ef368bc83e960c3d827e4a8d279

**Pull request merged**